client/src/components/RelatedProducts/ProductCard.jsx

```diff
@@ -130,7 +129,11 @@ const ProductCard = (props) => {
   return productCardPhoto && productCardInfo && (
     <React.Fragment>
       <Card className={classes.root} variant="outlined">
-        <Icon onClick={openShowComparison} sx={{ fontSize: 10 }} className={classes.icon}>grade</Icon>
+        <Grid container>
+          <Grid item align="right" xs={12}>
+            <Icon onClick={openShowComparison} sx={{ fontSize: 10 }} className={classes.icon}>grade</Icon>
```

Comment:
heh! I'm so glad I saw this... I've been using the native <i> tag... I will refactor these tomorrow!